Include/interactivity.h

```diff
@@ -237,6 +237,13 @@ class interactive_participant
 enum class interactive_event_type
 {
     /// <summary>
+    /// The message is not a message processed by the SDK. It could be a custom
+    /// message from an interactive control or a message supported by the protocol
+    /// but not yet supported by the SDK.
+    /// </summary>
+    unknown,

```

Comment:
So I thought about doing "custom" but it's not really custom. But "unknown" really is the most accurate. What do you think?